Source/KSCrash/Recording/Tools/KSDynamicLinker.c

```diff
@@ -272,6 +273,12 @@ bool ksdl_dladdr(const uintptr_t address, Dl_info* const info)
 
             for(uint32_t iSym = 0; iSym < symtabCmd->nsyms; iSym++)
             {
+                // Skip all debug N_STAB symbols
+                if ((symbolTable[iSym].n_type & N_STAB) > 0) 
```

Comment:
NIT: We actually want this to be zero, so this is a bit more specific.
```suggestion
                if ((symbolTable[iSym].n_type & N_STAB) != 0) 
```